package.json

```diff
@@ -5,8 +5,10 @@
   "scripts": {
     "start": "docusaurus start",
     "build": "docusaurus build",
-    "lint": "npm run lint:links && npm run lint:spelling && npm run lint:style",
-    "lint:links": "remark -f -q -u validate-links \"docs/**/*.mdx\"",
+    "lint": "npm run lint:spelling && npm run lint:links && npm run lint:style",
```

Comment:
Good call on changing the order. Maybe put links at the end so prettier will fail before it does the external requests as part of `linkinator`?

Reply:
Moved prettier to before links!